vNext/src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -20,4 +19,7 @@
 [assembly: InternalsVisibleTo("OrleansTestingHost")]
 [assembly: InternalsVisibleTo("Orleans.NonSiloTests")]
 [assembly: InternalsVisibleTo("TestInternalGrains")]
-[assembly: InternalsVisibleTo("TestExtensions")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("TestExtensions")]
+[assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("TesterInternal")]
+[assembly: InternalsVisibleTo("Tester.AzureUtils")]
```

Comment:
@xiazen it will compile even if it's gibberish:
![image](https://cloud.githubusercontent.com/assets/203839/20543364/e70ab9e8-b159-11e6-9cb6-13181464a470.png)